companies/management/commands/sync_corp_codes.py

```
@@ -122,6 +128,302 @@ def add_arguments(self, parser):
+        IndustryMapping 테이블 없이 메모리에서 직접 계산
+        """
+        self.stdout.write(self.style.SUCCESS("\n" + "=" * 50))
+        self.stdout.write("KIS 마스터 및 종목-업종 매핑 로드 중...")
+        self.stdout.write("=" * 50)
+
+        base_url = "https://new.real.download.dws.co.kr/common/master/"
+
+        # STEP 1: KisIndustry 테이블 채우기
+        self.stdout.write("STEP 1: KIS 업종 마스터 적재 중...")
+        if not dry_run:
+            res_idx = requests.get(base_url + "idxcode.mst.zip")
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)